lib/movable_ink/aws/autoscaling.rb

```diff
@@ -44,33 +44,59 @@ def delete_role_tag(role:)
         end
       end
 
-      def complete_lifecycle_action(lifecycle_hook_name:, auto_scaling_group_name:, lifecycle_action_token:)
-        run_with_backoff do
-          autoscaling.complete_lifecycle_action({
-            lifecycle_hook_name:     lifecycle_hook_name,
-            auto_scaling_group_name: auto_scaling_group_name,
-            lifecycle_action_token:  lifecycle_action_token,
-            lifecycle_action_result: 'CONTINUE'
-          })
+      def complete_lifecycle_action(lifecycle_hook_name:, auto_scaling_group_name:, lifecycle_action_token: nil, instance_id: nil)
```

Comment:
Any reason we'd want to pass instance_id as a parameter vs. using the `instance_id` method to retrieve the local instance ID?

Reply:
I didn't want to change the behavior or really rock the boat on what this is already doing, if we use the local instance id we'll never have need for the `lifecycle_action_token`.